recipe/conda_build_config.yaml

```diff
@@ -376,7 +376,7 @@ libkml:
 libmatio:
   - 1.5
 libnetcdf:
-  - 4.6
+  - 4.4
```

Comment:
Did you mean 4.5?

Reply:
Nope. I decided to go with 4.4 b/c of a mismatch on Windows Python 2.7, there is no 4.5 there due to modern c++. We patched 4.6 to make that happen but it would be a lot of work to backport those patches to 4.5.

The main issue will be installing `nco` in the same env as `netcdf4` b/c `nco` does require `4.5`. So for now people that need `nco`'s latest features will have to create separate envs.

Reply:
Reading https://github.com/Unidata/netcdf4-python/issues/836 looks like it is something we can "fix" by setting a higher timeout. If that is possible I'd rather build a new `4.6.1` than re-pinning and re-build all the stack.

@dopplershift what do you think?